From 66b4bda5f8faf199ea0fd46a50d9d6c21acea8cb Mon Sep 17 00:00:00 2001
From: Marc Lapierre <marc@eternallabs.com>
Date: Thu, 17 Oct 2013 15:14:08 -0400
Subject: [PATCH 40/61] JI:523296 CI:nschultz@qnx.com Updating
 car.profile.updateProfile to update PPS values if the
 modified profile is the active profile

---
 plugin/com.qnx.car.profile/src/blackberry10/profile.js |    5 +++++
 1 file changed, 5 insertions(+)

diff --git a/plugin/com.qnx.car.profile/src/blackberry10/profile.js b/plugin/com.qnx.car.profile/src/blackberry10/profile.js
index 125eb92..0e66820 100755
--- a/plugin/com.qnx.car.profile/src/blackberry10/profile.js
+++ b/plugin/com.qnx.car.profile/src/blackberry10/profile.js
@@ -155,6 +155,11 @@ module.exports = {
 		if (params.length > 0) {
 			_db.query(query.format(params.join(','), profileId));
 		}
+
+		//if active profile, update pps
+		if (profileId == this.getActive().id) {
+			this.setActive(profileId);
+		}
 	},
 
 	/**
-- 
1.7.10.msysgit.1

